src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/Streaming/Streaming.tsx

```diff
@@ -0,0 +1,150 @@
+import { FieldArray, useField } from 'formik';
+import { nanoid } from 'nanoid';
+import React, { useCallback, useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Button from '~core/Button';
+import { FormSection } from '~core/Fields';
+import Icon from '~core/Icon';
+import { CollapseExpandButtons } from '~dashboard/ExpenditurePage/Payments';
+import { Colony } from '~data/index';
+
+import { newFundingSource } from './constants';
+import { Streaming as StreamingType } from './types';
+import FundingSource from './FundingSource';
+import styles from './Streaming.css';
+
+const MSG = defineMessages({
+  fundingSource: {
+    id: 'dashboard.ExpenditurePage.Streaming.fundingSource',
+    defaultMessage: 'Funding source',
+  },
+  title: {
+    id: 'dashboard.ExpenditurePage.Streaming.title',
+    defaultMessage: '{nr}: {team}',
+  },
+  team: {
+    id: 'dashboard.ExpenditurePage.Streaming.team',
+    defaultMessage: 'Team',
+  },
+  rate: {
+    id: 'dashboard.ExpenditurePage.Streaming.rate',
+    defaultMessage: 'Rate',
+  },
+  addFundingSource: {
+    id: 'dashboard.ExpenditurePage.Streaming.addFundingSource',
+    defaultMessage: 'Add funding source',
+  },
+  notSet: {
+    id: 'dashboard.ExpenditurePage.Streaming.notSet',
+    defaultMessage: 'Not set',
+  },
+});
+
+const displayName = 'dashboard.ExpenditurePage.Streaming';
```

Comment:
Same here - missing `ExpenditureSettings`

Reply:
I moved "Streaming" folder to "ExpenditurePage". I created it in "ExpenditureSettings" by mistake. Thank you for pointing it.